### DISCUSSION TITLE
Looks like Riot are banning Linux users now. Perhaps add a notice to the readme.

### DISCUSSION THREAD

**Discussion by ghost** [M-Reimer/wine-lol#121] [Announcements]
Had multiple accounts banned within a few ranked matches for 'cheating' (as I do not use other tools it has to be Linux related). The accounts have been fine in previous patches. Consider adding a notice on the readme and perhaps others can confirm this too.

**Comment by M-Reimer** [comment]
I've opened a discussion section here and migrated your issue.

I stopped playing years ago, so not much I can do here. But maybe others can join the discussion.

Did you contact Riot support already?

**Comment by ghost** [comment]
> Did you contact Riot support already?

Yeah I did. Will see what their response is. I did however come across this this tweet https://twitter.com/RiotK3o/status/1682122977050238980 after opening this issue and it's plausible that is the cause rather than Wine or Linux. I didn't dare to test on main account when I noticed alt accounts were receiving bans and as Riot have never cared about bot-leveled accounts in the past (and the fact that these have been fine for months) it seemed unlikely to be the cause. However if Riot are doing a year's worth of retroactive bans to botted accounts then that seems more probable than wine or linux being the culprit. 

You can close this if you like but it'd be nice if someone else could confirm it is in fact not wine or linux related.

**Reply by Exellin** [reply]
Did you ever get a response from Riot?  I just got a message saying I was permanently banned with no warning or anything.

**Comment by M-Reimer** [comment]
This is a discussion and no issue. So I don't see any reason why to close it. Keep us updated.
